src/main/java/strategy/ReturnOperation.java

```
@@ -0,0 +1,11 @@
+package strategy;
+
+import db.Storage;
+import model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    @Override
+    public void apply(FruitTransaction transaction) {
+        Storage.addQuantity(transaction.getFruit(), transaction.getQuantity());
```

The code does not catch or rethrow exceptions with additional context. The project requires: "Do not ignore exceptions; rethrow as RuntimeException with informative messages and original cause." If `Storage.addQuantity(...)` can throw, catch the exception, wrap it in a RuntimeException that includes transaction details (operation, fruit, quantity) and the original cause, then rethrow so failures are easy to trace back to the CSV row/transaction.